packages/client/components/GitLabScopingSearchResults.tsx

```diff
@@ -78,7 +59,14 @@ const GitLabScopingSearchResults = (props: Props) => {
   >(
     graphql`
```

Comment:
The query is a little long and could be moved to its own file, but I've kept it here to be consistent with the GitHub logic, and so that it's easily accessible